www/index.php

```diff
@@ -703,9 +705,14 @@ function onRecaptchaSubmit(token) {
                                             See <a href="https://sites.google.com/a/webpagetest.org/docs/using-webpagetest/custom-metrics">the documentation</a> for details on how to specify custom metrics to be captured.
                                         </div></div>
                                     </div>
-                                    
+
                                     <p><label for="custom_metrics" class="full_width">Custom Metrics:</label></p>
                                     <textarea name="custom" id="custom_metrics" cols="0" rows="0"></textarea>
+
+                                    <?php if (isset($_REQUEST['heroElementTimes']) && $_REQUEST['heroElementTimes']): ?>

```

Comment:
If we are exposing it in the UI could you also create a markdown doc in the docs folder with information and examples and then link to it from here?  I'd be ok with exposing it through the API with no docs but if we add it to the UI we need to explain to people how to use it.

Reply:
Of course! Is a separate yellow box like this okay?
![screenshot_2018-07-04 webpagetest - website performance and optimization test 1](https://user-images.githubusercontent.com/794263/42245367-a3de9f94-7f6c-11e8-9654-d6da9c04e3d1.png)